src/dialects/abstract/query-generator-typescript.ts

```diff
@@ -39,6 +39,14 @@ export class AbstractQueryGeneratorTypeScript {
     return this.sequelize.options;
   }
 
+  describeTableQuery(table: TableNameWithSchema) {
+    return `DESCRIBE ${this.quoteTable(table)};`;
+  }
+
+  showIndexesQuery(_table: TableNameWithSchema): string {
```

Comment:
same here, this should accept `TableName` (or even `TableNameOrModel`) and normalize using `extractTableDetails`